src/main/resources/de/tracetronic/jenkins/plugins/ecutestexecution/steps/ProvideUnitReportsStep/config.jelly

```diff
@@ -5,10 +5,10 @@
         <b>${%step.compatNote}</b>
     </f:description>
     <f:entry title="${%unstableThreshold.title}" description="${%unstableThreshold.description}" field="unstableThreshold" >
-        <f:number min="0" max="100" default="0" />
```

Comment:
i found the implementation as number good and it works too. The Check validates the number, the user can use the provided buttons or could type the number as text if needed